app/pages/patients/patients.js

```diff
@@ -98,7 +94,7 @@ export let Patients = React.createClass({
     var handleClickYes = function(e) {
       e.preventDefault();
       self.props.onHideWelcomeSetup();
```

Comment:
But yea ok, if you do click yes and then close tab (and have remember me clicked) you won't see the welcome message again, but I'd argue thats your fault - you acknowledged the message, and we do have the fall back message anyway that gives affordance on what to do without anything set up.